src/interfaces.ts

```diff
@@ -198,6 +198,11 @@ export interface WalletConnectOptions extends CommonWalletOptions {
   bridge: string
 }
 
+export interface MewConnectOptions extends CommonWalletOptions {
+  rpcUrl: string,
+  networkId: number
```

Comment:
@gamalielhere `networkId` is already a part of `CommonWalletOptions` so we should remove it from here